crates/astria-sequencer/src/accounts/state_ext.rs

```diff
@@ -0,0 +1,72 @@
+use anyhow::Result;
+use async_trait::async_trait;
+use penumbra_storage::{
+    StateRead,
+    StateWrite,
+};
+
+pub type Address = String;
+pub type Balance = u64; // might need to be larger
```

Comment:
Please avoid type aliases for primitive types because it makes it difficult to parse and understand code (and figure out what's wrong in compiler errors).

Type aliases are generally used for simplifying complex types, for example `pub type Nested<Item> = N<N<N<N<N<N<N<N<L<Item>>>>>>>>>;` (from the penumbra codebase).

I think these types are prime candidates for newtype wrappers, for which you can either serde or borsh. You can then get rid of the entire parsing logic.

For serde, have a look at bincode.

For borsh: https://borsh.io/

Reply:
I updated these and put them in `account/types.rs` and used borsh for encoding/decoding before putting in storage. let me know what you think!